include/zephyr/mgmt/mcumgr/grp/os_mgmt/os_mgmt_client.h

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef H_OS_MGMT_CLIENT_
+#define H_OS_MGMT_CLIENT_
+
+#include <inttypes.h>
+
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+struct smp_client_object;
+
+/**
+ * @brief OS mgmt client object
+ */
+struct os_mgmt_client {
+	/* SMP client object  */
```

Comment:
```suggestion
	/** SMP client object  */
```

Same in this header. Doxygen blocks start with `/*` instead of `/**`